packages/components/molecules/f-mega-modal/package.json

```diff
@@ -49,11 +49,11 @@
   ],
   "peerDependencies": {
     "@justeat/browserslist-config-fozzie": ">=1.1.1",
-    "@justeat/f-button": "3.x",
+    "@justeat/f-button": "4.x",
     "body-scroll-lock": "3.x"
```

Comment:
Do we need to make this change in the devDep too?

Reply:
It was already at 4.x 👍

Reply:
As mentioned, this PR only focuses on JE packages.